js/darkmode.js

```diff
@@ -0,0 +1,45 @@
+onload = onload();
+const icon = document.getElementById('icon');
+icon.onclick = function () {
+  const element = document.body;
+  element.classList.toggle('dark-theme');
+  if (element.classList.contains('dark-theme')) {
+    icon.src = 'img/sun.png';
+    setCookie('theme', 'dark', 30);
+  } else {
+    icon.src = 'img/moon.png';
+    setCookie('theme', 'light', 30);
+  }
+};
+
+function onload() {
+  const themeIcon = document.getElementById('icon');
+  if (getCookie('theme') === 'light') {
+    document.body.classList.remove('dark-theme');
+    themeIcon.src = 'img/moon.png';
+  } else if (getCookie('theme') === 'dark') {
```

Comment:
No need of `else if` here. We can directly go with else.

Reply:
Just wanted to be more specific for that cookie name.

Reply:
@rohan09-raj Store the value returned from the `getCookie` function in a variable and use it for comparision rather than calling it twice.